handlers/main.yml

```diff
@@ -3,3 +3,4 @@
   service:
     name: docker
     state: reloaded
+  when: not nvidia_docker_skip_docker_reload
```

Comment:
Could we make it automatic to avoid adding an extra variable?
```yaml
- name: Populate service facts
  ansible.builtin.service_facts:

- name: reload docker
  ...
  when: 'docker' in ansible_facts.services
```

Reply:
This requires fixes from an ansible version newer than what we install with deepops: https://github.com/ansible/ansible/issues/71140

Reply:
Rats! Nvm then.